### ISSUE TITLE
Merge doc makefiles into a Python script

### ISSUE THREAD

**Issue by datapythonista** [python-sprints/pandas-mentoring#96]
In the `doc/` directory we've got two different scripts, that were created by Sphinx, `Makefile` and `make.bat`. The reason why Sphinx does this is because a `Makefile` only works in unix system (Linux or Mac mainly), so a separate `make.bat` file to build the docs in windows is provided.

But instead of having two separate scripts, seems more efficient to have a single script in a language that works in any platform. Python looks like a good option for that. Can we replace those files by a Python file that builds the documentation and provides the same functionality?

**Comment by Ayowolet** [comment]
I'll take this one

**Comment by sparalic** [comment]
@Ayowolet  do you need help with this issue?

**Comment by Ayowolet** [comment]
@sparalic no thank you... was distracted by the azure pipelines...

**Comment by Ayowolet** [comment]
@datapythonista I'm struggling with this a little bit. Can I use the make.py file in the pandas repository as a guide?

**Comment by datapythonista** [comment]
Absolutely, we need something much simpler here, but everything we need should be in that file.

Feel free to open a PR that is not finished/not working, so people can have a look and help you.

**Comment by Ayowolet** [comment]
Okay, thanks. I will do that.

**Comment by Ayowolet** [comment]
I've taken too long on this.. @sparalic if you want to work on this, please feel free.. I've not been able to devote as much time as I ought to.. @datapythonista I apologise for spending too long

**Comment by sparalic** [comment]
@Ayowolet sure I can help you with this one. @datapythonista do you want this copied over from the pandas repo, or should I use that as a working doc and retrofit it to our needs?

**Comment by datapythonista** [comment]
The pandas one is very complex. You can use it for inspiration, but I'd just create a very simple one with what the script are doing now, but in Python, so a single script work in any OS.
